index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sameed - Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+</head>
+
+
+
+<body>
+
+    <header>
+        <nav>
+            <div class="logo-container">
+                <img src="images for portfolio/logo.jpg" alt="" class="logo">
```

Comment:
@sameedmohsin28 As per the content of your portfolio, I believe that you are using `Template 1` of FIGMA.
- So, the purpose of this project is to strictly follow FIGMA design.
- There is no image logo in this design.
![image](https://user-images.githubusercontent.com/61361037/212485874-faa35693-5001-4a6c-b63d-6e272b617abb.png)

Kindly Update the `font-size`.